README.md

```diff
@@ -66,6 +66,7 @@ By default, all variables are assumed to be `true`.
 vars:
   using_twilio_call: False # Disable this if not using call
   using_twilio_messaging_service: False # Disable this if not using messaging_service
+  using_twilio_usage_record: False # Disable this if not using call
```

Comment:
This needs to be updated to reference the appropriate source table for this variable.
```suggestion
  using_twilio_usage_record: False # Disable this if not using usage_record
```

Reply:
thanks for catching! updated